demo/config.js

```diff
@@ -480,8 +480,9 @@ shakaDemo.Config = class {
         .addBoolInput_('Allow Media Source recoveries',
             'streaming.allowMediaSourceRecoveries')
         .addNumberInput_('Minimum time between recoveries',
-            'streaming.minTimeBetweenRecoveries');
-
+            'streaming.minTimeBetweenRecoveries')
+        .addBoolInput_('Use Actual Live Stream Duration',
+            'streaming.useActualLiveStreamDuration');
```

Comment:
I think this should be renamed.  Maybe something like `infiniteLiveStreamDuration` would be clearer.

Reply:
Renamed `infiniteLiveStreamDuration`